cli/omnibus/config/software/kontena.rb

```diff
@@ -7,6 +7,7 @@
 dependency "libxslt"
 whitelist_file "./wrappers/sh/kontena"
 build do
+  gem "install rb-readline -v 0.5.4 --no-ri --no-doc"
```

Comment:
Also the omnibus thing should be able to produce windows installers if you run it on windows. Rb-readline README says that one of the reasons it was written is because readline is nearly impossible to build on windows. (kosh seems to work fine on win10 and rubyinstaller installed ruby, so not sure if relevant)